### ISSUE TITLE
[Suggestion] Add Player Selling Back

### ISSUE THREAD

**Issue by MokahTGS** [blay09/FarmingForBlockheads#17]
Can you possibly add a way for players to sell back crops/seeds/saplings/etc so that we can have a one stop shop for a market.  I'd even support a bit of a loss in emeralds for the player selling back farm products.  The merchant needs to make a living somehow.  :)

**Comment by arfink** [comment]
I second this, although I personally think it would be cool to mirror the Pam's / Harvest Moon theme, and make a separate shipping bin to load produce into, and then when you push a button on the produce bin (or whatever it happens to be) then another villager will plummet from the sky, open the chest, and take your stuff, leaving behind emeralds.

**Comment by MokahTGS** [comment]
And then please helecoppter back up into the air

**Comment by BedrockLegends** [comment]
I agree with this if there is away for Modpack creators to make whatever their selling based on item or price configurable as most players will take advantage of this feature if what they are selling will give them an item like a diamond

**Comment by joe-vettek** [comment]
Hei,I think it's a great idea,and give a insteresting way to get something for some player.Is it work well now?I saw someone say that there's something wrong with it.

**Comment by JustinTheBob** [comment]
You should be able to sell seeds back. Just write it in the market registry. Make the seed you wanna "sell back" the "item" and make the item you want to get from selling the seed back the "payment". I've added a couple items you can sell him for gold, iron etc in return.

**Comment by whiteeagle1985** [comment]
Maybe if there was a way to limit sell backs per period of time so it doesn't become overpowered, then I'd approve of it. 

I'd like to avoid this situation: I played a pack where they used KubeJS to sell cooked items for emeralds via the crafting table and with automation, I had a steady stream of emeralds and emerald blocks to buy whatever I needed from the Market. Might not have been early game automatable, but it was still possible once I had Create producing items I needed on demand.

**Comment by BlayTheNinth** [comment]
Added in Minecraft 26.1+
